client.go

```diff
@@ -480,3 +484,82 @@ ExitForLoop:
 		}
 	}
 }
+
+func runHTTPLatencyTest(test *ethrTest) {
+	uri := test.session.remoteAddr
+	ui.printMsg("hi")
```

Comment:
Please remove this comment and convert next statement to printDbg.

Reply:
Done